diagrams/details.svg

```diff
@@ -0,0 +1,3 @@
orm="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 178px; height: 1px; padding-top: 119px; margin-left: 593px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font face="Verdana">Name of the Ethereum network this project is for.</font></div></div></div></foreignObject><text x="682" y="122" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Name of the Ethereum network t...</text></switch></g><path d="M 530 182.5 L 581.9 182.5" fill="none" stroke="#ff0000" stroke-width="3" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 588.65 182.5 L 579.65 187 L 581.9 182.5 L 579.65 178 Z" fill="#ff0000" stroke="#ff0000" stroke-width="3" stroke-miterlimit="10" pointer-events="all"/><rect x="30" y="160" width="500" height="45" fill="none" stroke="#ff0000" stroke-width="3" pointer-events="all"/><rect x="592" y="142.5" width="490" height="80" fill="none" stroke="#ff0000" stroke-width="3" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 488px; height: 1px; padding-top: 182px; margin-left: 593px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font face="Verdana">The RPC URL that you can use to send your transactions to FuelStop, as well as other standard Ethereum JSON-RPC requests. Note that there is a unique project ID for you at the end of the URL, which is used to authenticate your RPC calls. You are supposed to keep this as a secret, and never share it with anyone else.</font></div></div></div></foreignObject><text x="837" y="186" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">The RPC URL that you can use to send your transactions to FuelStop, as well as oth...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
The RPC URL that you can use to send your transactions to FuelStop. You can also use it to send standard Ethereum JSON-RPC requests. Note that your unique project ID part of the URL at the end. This project ID is used to authenticate your RPC calls. You MUST keep this a secret, and never share it with anyone else